### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by GrapeApe230** [advplyr/audiobookshelf#171] [General]
Where's your donate button? I'll throw some money/crypto at this.

**Comment by advplyr** [comment]
Update: [Github sponsors](https://github.com/sponsors/advplyr) allows one-time and recurring donations. Thanks!

I haven't setup a donation button. The project is still young and there are a lot of contributors helping out already, which is awesome. The best way to support the project right now is to help test new features, provide feedback, come up with better designs, improve the documentation, or open a pull request if you are a developer.
Just using/starring/following the project is support enough, I appreciate it!

**Reply by advplyr** [reply]
I should probably add a "how to support the project" section somewhere.

**Comment by Helico-Pter** [comment]
You've done a great job with this project already and I'd be happy to contribute when you're ready to accept donations.

**Reply by zombiehoffa** [reply]
I  agree with both of you. I would donate once the android app gets a lot less buggy, clicking download works 100% of the time, viewing and keeping track of position in offline library works 100% of the time, and syncing play location is a lot more robust so that whether I'm listening to a local copy offline, streaming to the app, or listening on the server I can be pretty sure that once I reconnect the client it is highly likely my position will be correct everywhere. in other words, it's not quite mature enough yet to get a donation (but I am super excited about it getting there! The concept is awesome!)

**Comment by Jdiesel87** [comment]
Do you have a long term financial plan for the project?

My preference is for projects where the server backend and webplayer remain free and the apps are sold in the appstores as one time payments. I believe that developers should be compensated for their work, other freeloads expect everything to be free.

**Reply by advplyr** [reply]
I have no plans to monetize the project. I would like to keep everything open source. I think the best thing right now is to encourage more people to contribute. The documentation could really use some help, plugins for other services, a Windows installer, etc.

**Reply by Jdiesel87** [reply]
Well make sure you get a donation system setup asap, it would be a shame to not get some compensation for your hard work.

**Comment by feerlessleadr** [comment]
Just want to echo that you should throw up a donate button. This app is something I've wanted for so long, and you (and others) have brought it a long way.

**Comment by andrewdunndev** [comment]
This software has a big impact on peoples lives, it fills a niche that nothing else filled well before. I think many would donate to show support.

**Comment by rweatherly** [comment]
i have been looking for ways to donate to the project as well. i admire the commitment to open source, but project burn out is a real thing. you can commit to full open source and in the future and maybe offer minor add-ons like way to have hosted proxy for remote access for people or something. I'm sure there would be a TON of technical and legal work in making that a real thing so I can understand not wanting to do it. 
i will be trying to put together some feedback as well as this is the single most important application i self-host, for myself and close family members. I just want to make sure it doesn't go away in the long term.

**Comment by xppx99** [comment]
I would like to donate, this software is amazing.

**Comment by kieraneglin** [comment]
I would also like to donate! Setting up a patreon or paypal would be great

**Comment by jasonmhite** [comment]
I'd like to donate, I love this software.

**Comment by darthrater78** [comment]
Same

**Comment by KaiStarkk** [comment]
Does anyone know if Feature Bounties are still a thing in the FOSS community?
That would be a cool way to support contributors while also helping to fund features you care about.

**Comment by advplyr** [comment]
I finally got around to setting up a [Github sponsors profile](https://github.com/sponsors/advplyr) that can take one-time and recurring donations.

Currently this is under my personal Github account but will probably be setup in the Abs organization account once the repos get moved over there. The only downside is that Github takes 6% of the donations from organizations and doesn't take anything from personal accounts.

The best way to support the project is still to contribute with designs, code, docs, and helping out other users in Discord/Matrix/Github/etc. but not everyone has the time/resources for that.

Thanks to everyone asking about donations over the last 2 years!

https://github.com/sponsors/advplyr

**Reply by rweatherly** [reply]
Thank you for setting that up, i have been a long-time user, and have tried learning a few things to contribute code back, but it's a bit beyond me skill wise, so I'm in for a monthly sponsorship!
this is one of my favorite GH projects and one i use literally every day.

**Reply by projx** [reply]
I've used ABS since you first announced it on r/selfhosted, its single piece of software I consider "critical" in my personal life...  My coding-fu is a bit meh, but I will take a look once I wrap up my current job. But in the meantime, thank you for all your efforts (and same to all contributors), a monthly donation is the absolute minimum this project derserves.  

On another note, it may be worth mentioning this on the How To Support page: https://www.audiobookshelf.org/support

**Comment by salixh5** [comment]
There are people running ABS through https://www.pikapods.com/ they have a revenue sharing thing going on where you could easily get a part of their ABS-related income.
